app/src/main/java/com/iic/shopingo/ui/OnboardingActivity.java

```diff
@@ -17,16 +20,19 @@
 
   private static final String LOG_TAG = OnboardingActivity.class.getSimpleName();
 
-  @InjectView(R.id.onboarding_login_btn)
-  LoginButton facebookLoginBtn;
-
   private UiLifecycleHelper facebookLifecycleHelper;
 
+  @InjectView(R.id.onboarding_login_btn)
+  LoginButton loginBtn;
+
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_onboarding);
 
+    ButterKnife.inject(this);
+
+    loginBtn.setReadPermissions("public_profile", "user_location");
```

Comment:
Couldn't find any and in their docs and tutorial they don't use constants just regular strings.